CourseGrab/Course.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Course.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 1/26/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+enum Status: String, Codable {
+    case open, closed
+}
+
+class Course: Codable {
```

Comment:
is there a reason this needs to be a `class` instead of just a `struct`?
if you made it a struct you. also wouldnt need the long init